alchemy-web/docs/providers/sentry/client-key.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Managing Sentry Client Keys with Alchemy
+description: Learn how to create, configure, and manage Sentry client keys using Alchemy.
+---
+
+# ClientKey
+
+The ClientKey resource lets you create and manage [Sentry](https://sentry.io) client keys for your projects.
+
+## Minimal Example
+
+Create a basic Sentry client key:
+
+```ts
+import { ClientKey } from "alchemy/sentry";
+
+const key = await ClientKey("my-key", {
+  name: "My Key",
+  project: "my-project",
+  organization: "my-org"
+});
+```
```

Comment:
The docs are missing how to auth (`SENTRY_AUTH_TOKEN` env or pass in `apiToken`).

Reply:
Resolved via 0e344d425a966b5a44e76283744a1f5f030399f7